### PR TITLE
Minor cleanup

### PR DESCRIPTION
**Pull request by jakirkham**
* Use PyPI for stable checksums.
* Package the license file.

### CODE REVIEW

**Comment by jreback** [comment]
thanks @jakirkham

**Comment by jorisvandenbossche** [comment]
Commented in the other PR:

Well, then we get the complaint that the PyPI source package is available before release time .. https://github.com/pandas-dev/pandas/issues/16260

Unless we first build all wheels, and only then conda-forge packages, but then the conda packages are only available later

**Comment by jorisvandenbossche** [comment]
The change in github checksum is our fault. We put a wrong link in the announcement, and after some back-and-forth corrected after everything was built.. But forgot that conda-forge packages might have to be rebuilt

**Comment by jakirkham** [comment]
I see. Well I would recommend publishing a tarball under the GitHub release in the future. There are a number of issues with GitHub provided archives. Publishing your own tarball under the release should avoid these issues.

**Comment by jakirkham** [comment]
How would you like to resolve this in the interim? Would as that publishing a release on GitHub could still be an option.

**Comment by jakirkham** [comment]
> The change in github checksum is our fault.

TL;DR GitHub archives can change checksum with no intervention of your own. Hence it is better to upload your own tarball.

It's actually possible for the GitHub archive's checksum to change without any intervention on your part. The thing is GitHub archives (i.e. the source code links below each tag) are dynamically generated each time a user clicks on them. While in theory the checksum should never change (as the generation process should be identical), for whatever reason it does change. This is because GitHub may have changed something like the date time on the files, the user that generated it, what order the files got added to the archive, etc. The result is the checksum appears to have changed; even though, the tag itself has not been modified.

To resolve this, one can upload their own tarball under the GitHub release (recommended by GitHub). This artifact will be static and so will have a fixed checksum. Using Travis, it is possible to [automate]( https://docs.travis-ci.com/user/deployment/releases/ ) this process.

**Comment by jorisvandenbossche** [comment]
Ah, didn't know that. Then I think we can add that to our release process to manually upload an archive to github. 

cc @TomAugspurger

**Comment by jorisvandenbossche** [comment]
Something else: would it make sense to rebuild 0.19.2 with numpy 1.13 as well?  
0.19.2 is currently still more downloaded (last week) than 0.20.2

**Comment by jakirkham** [comment]
That's up to you, but it certainly can be done.

**Comment by jreback** [comment]
@jakirkham this looks good. We could always come back with a recipie push to update 0.19.2 (later)

**Comment by jreback** [comment]
thanks @jakirkham

**Comment by jakirkham** [comment]
np

**Comment by jakirkham** [comment]
> Something else: would it make sense to rebuild 0.19.2 with numpy 1.13 as well?
> 0.19.2 is currently still more downloaded (last week) than 0.20.2

Probably not the whole reason, but found that [`calliope`]( https://github.com/conda-forge/calliope-feedstock ) is restricting `pandas` to 0.19.x. Maybe there are other cases like this?

**Comment by jreback** [comment]
well is someone wants to upgrade numpy but not pandas is kind of odd
